redbot/cogs/trivia/data/lists/whosthatpokemon.yaml

```diff
@@ -1,4 +1,5 @@
 AUTHOR: aikaterna
+DESCRIPTION: Can you identify the pokémon from their image?
```

Comment:
To my knowledge, each of the `whosthatpokemon` trivia lists is about a specific Generation of pokemons, i.e. `whosthatpokemon` is about Generation I, `whosthatpokemon2` is about Generation II, and so on. It would be good if this description included information about the generation it's about.

Reply:
> To my knowledge, each of the `whosthatpokemon` trivia lists is about a specific Generation of pokemons, i.e. `whosthatpokemon` is about Generation I, `whosthatpokemon2` is about Generation II, and so on. It would be good if this description included information about the generation it's about.

I see, I wasn't aware of this so thanks for letting me know. I'll make those changes